datamade/la-metro-dashboard
# Containerize remaining DAGs

### Description

This PR:

- Adapts the remaining DAGs to run in containers, per the approach outlined in #32. Closes #30.
- Cleans up non-containerized artifacts (specifically, removes `base.py` and removes installation of Metro and the scrapers in the dashboard image).
- Pulls DAG descriptions into an object in `constants.py`.
- Refactors `get_db_info()` in the dashboard plugin to hit a new object counts endpoint from the Metro app.

### Testing instructions

- Rebuild the dashboard to confirm the updated Dockerfile works as expected.
- Trigger `refresh_guid`, `windowed_bill_scraping`, and `windowed_event_scraping` and confirm they complete successfully.
- Trigger `friday_hourly_scraping` and `saturday_hourly_scraping` and confirm the scrapes begin. You can terminate the runs when you start to see scrape logs.
- Double check that I didn't screw up any of the cron scheduling strings during the transition. 😅
- Navigate the dashboard and confirm you see the correct counts for your local Metro install.
- Navigate to the staging site dashboard and confirm you see the correct counts for the Metro staging site.
